docs/_docs/testnet/starting-a-node.md

```diff
@@ -0,0 +1,49 @@
+---
+---
+# Starting a node
+How to start a node in the Evan's Cat testnet.
+
```

Comment:
Should you state that this is for a validating node only?

Reply:
I don't believe so. External users can only run validators.